install.sh

```diff
@@ -15,61 +13,52 @@ apt-get --yes install epoptes epoptes-client
 
 
 #Adding vagrant user to group epoptes
-
 gpasswd -a ${SUDO_USER:-$USER} epoptes
 
 #Updating kernel
-
 echo 'IPAPPEND=3' >> /etc/ltsp/update-kernels.conf
 /usr/share/ltsp/update-kernels
 
 #configure dnsmasq
-
 ltsp-config dnsmasq
 
 #Creating lts.conf
-
 ltsp-config lts.conf
 
 #Installing additional software
-
 apt-get --yes install edubuntu-desktop
 apt-get --yes install ubuntu-edu-preschool ubuntu-edu-primary ubuntu-edu-secondary ubuntu-edu-tertiary
 
 #Installing ltsp-manager
-
 add-apt-repository ppa:ts.sch.gr
 apt-get --yes update
 apt-get --yes install ltsp-manager
 
 #Creating client image
-
 ltsp-update-image --cleanup /
 
-#setting correct dhcp-range
-sed -i "s/dhcp-range=192.168.67.20,192.168.67.250,8h/dhcp-range=192.168.1.20,192.168.1.250,8h/g" /etc/dnsmasq.d/ltsp-server-dnsmasq.conf
-service dnsmasq restart	
+#source setting.sh
+source /vagrant/settings.sh
+NET=$(echo $NETWORK | cut -d'.' -f 3)
+
+#setting dhcp-range
+sed -i 's/dhcp-range=192.168.67.20,192.168.67.250,8h/dhcp-range=192.168.'"$NET"'.20,192.168.'"$NET"'.250,8h/g' /etc/dnsmasq.d/ltsp-server-dnsmasq.conf
 
-#Reading setting.sh file
-cd /vagrant
-read -a arr < settings.sh
-echo ${arr[0]} 
-var=${arr[0]}
-response=($(echo "$var" | tr '=' '\n'))
 
 #Setting mode of operation of ltsp server
-if [ "${response[1]}" == "Yes" ] || [ "${response[1]}" == "yes" ]; then
+if [ "$STANDALONE" == "Yes" ] || [ "$STANDALONE" == "yes" ]; then
 	echo "LTSP server will be in standalone mode of operation"	
```

Comment:
If you convert the value of `$STANDALONE` to lowercase (or uppercase), you can make only one test. For example:
```
if [[ ${STANDALONE,} == "yes" ]]; then
```
There are a lot of tricks that you can learn about bash scripting. It seems that you have not studied very carefully the book that I told you, or you have missed some important parts of it.

Reply:
>It seems that you have not studied very carefully the book that I told you, or you have missed some important parts of it.

sorry about that.
can I use `if [[ ${STANDALONE,,} == "yes" ]]; then`   instead
It can handle case of `YES` along with `Yes`.